modules/server/main.tf

```diff
@@ -113,6 +116,18 @@ data "template_file" "forseti_server_env" {
   }
 }
 
+data "template_file" "run_forseti" {
```

Comment:
How will this file get copied to the server? I think we'll need to update the startup script template to copy this from GCS.

Reply:
Updated. PTAL